plugin/core/rpc.py

```diff
@@ -87,13 +87,21 @@ def __init__(self, transport: Transport, settings) -> None:
     def send_request(self, request: Request, handler: 'Callable[[Optional[Any]], None]',
                      error_handler: 'Optional[Callable]' = None) -> None:
         self.request_id += 1
-        debug(' --> ' + request.method)
-        self._response_handlers[self.request_id] = (handler, error_handler)
-        self.send_payload(request.to_payload(self.request_id))
+        if self.transport is not None:
+            debug(' --> ' + request.method)
+            self._response_handlers[self.request_id] = (handler, error_handler)
+            self.send_payload(request.to_payload(self.request_id))
+        else:
+            debug(' sending ' + request.method + ' omitted')
```

Comment:
why did you prefix the word "sending" with a space here?

it might be best to use this:

```
debug('sending', request.method, 'omitted')
```
the `debug` function concatenates all of its arguments together with spaces in-between, just like the `print` function.

Reply:
Good point. I added the space because the other message is `debug(' --> '+...).`